packages/rx_bloc_cli/mason_templates/bricks/lib_social_logins/README.md

```diff
@@ -7,6 +7,12 @@ When you run the created project, don't forget to enable the "Sign in with Apple
 
 Supports iOS.
 
+Facebook authentication uses [flutter_facebook_auth][6] package. In order to make it work you must register your app in facebook developer console, there you will find your <strong>app_id</strong>, <strong>client_token</strong> and <strong>app_name</strong>,
```

Comment:
Please add a section in the main README.md file as well - > https://pub.dev/packages/rx_bloc_cli/example